dali/ft/filecopy.cpp

```diff
@@ -2662,6 +2662,16 @@ void FileSprayer::checkFilePath(RemoteFilename & filename)
             Owned<IConstDropZoneInfo> dropZone = env->getDropZoneByAddressPath(netaddress.str(), pfilePath);
             if (!dropZone)
                 LOG(MCdebugInfo, unknownJob, "No matching drop zone path to file path: '%s'", filePath.str());
+            else
+            {
+                SCMStringBuffer dropZoneName;
+                dropZone->getName(dropZoneName);
+                LOG(MCdebugInfo, unknownJob, "Drop zone path '%s' is %svisible in ECLWatch."
+                        , dropZoneName.str()
+                        , (dropZone->isECLWatchVisible() ? "" : "not ")
```

Comment:
@jakesmith 
No, it doesn't because if it is visible then it prints out an empty (0 length) string:
   0000000C 2016-09-20 11:01:25.360 25260 25271 "Drop zone path 'mydropzone' is visible in ECLWatch."

 If it false then there need a space after 'not' because the '%s' is prefix of the 'visible' text.
   0000000C 2016-09-16 11:35:40.288 31366 31377 "Drop zone path 'mydropzone' is not visible in ECLWatch."

Originally it was 
LOG(MCdebugInfo, unknownJob, "Drop zone path '%s' is %s in ECLWatch."
, dropZoneName.str()
, (dropZone->isECLWatchVisible() ? "visible" : "not visible"));
but I thought why put 'visible' twice?